frontend/public/components/markdown-view.tsx

```diff
@@ -17,14 +17,21 @@ type ShowdownExtension = {
   replace?: (...args: any[]) => string;
 };
 
-const markdownConvert = (markdown, extensions: ShowdownExtension[]) => {
+type ShowdownOption = {
```

Comment:
You should be able to use https://www.npmjs.com/package/@types/showdown to get the real types.

Reply:
Done.  Thanks for the assist!